### PR TITLE
Add a toggle to enable Silly Optimizer

### LINKED ISSUE #1112: Keep Silly Wisher assets as an optional toggle
image
image
image
image
image
image
image
imagev
image
image
image
image
image
image
image
image
image

### PR DESCRIPTION
**Pull request by frzyc**
Due to overwhelming demand, resolves #1112 
Partially setup #974 
Also unify the usage of the "character side icon"

image

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ jobs:
           yarn install --immutable --immutable-cache
       - name: Run lint
         run: |
-          yarn run nx run-many --target=lint --max-warnings=0
+          NODE_OPTIONS="--max-old-space-size=4096" yarn run nx run-many --target=lint --max-warnings=0
```
Comment (frzyc) [inline .github/workflows/ci.yml:29]: @eeeqeee I carried over this change. Let me know if you think this change should not go into master.

File: apps/frontend/src/app/PageSettings/SillyCard.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    <CardLight>
+      <CardContent sx={{ display: 'flex', alignItems: 'center', gap: 2 }}>
+        <Avatar src={silly_icon} />
+        <Typography variant="h5"> {t`sillyPageTitle`}</Typography>
+      </CardContent>
+      <Divider />
+      <CardContent>
+        <Typography>
+          <Trans t={t} i18nKey="settings:sillyCard.desc">
+            We <s>stole</s> borrowed(with permission) all the character assets
```
Comment (JoseZehi) [inline apps/frontend/src/app/PageSettings/SillyCard.tsx:31]:
```suggestion
            We <s>stole</s> borrowed (with permission) all the character assets
```

Space before `(`

File: libs/silly-wisher-names/src/loadTrans.ts [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+    es, // Spanish
+    ko, // Korean
+    th, // Thai
+    id, // indonesian
+    pt, // Portuguese(Brazil)
+    ru, // Russian
+    ,
+    // Polish
+    vi, // Vietnamese
+    tr, // Turkish // Arabic, Filipino / Tagalog
```
Comment (eeeqeee) [inline libs/silly-wisher-names/src/loadTrans.ts:125]: These comments are confusing. Perhaps use underscore prefixed variables instead, e.g. `_po`

**Review by eeeqeee — CHANGES_REQUESTED** [review]

File: libs/silly-wisher-names/src/loadTrans.ts (hunk 3 above)
Comment (frzyc) [reply libs/silly-wisher-names/src/loadTrans.ts:125]: yeah I had to fight with prettier, I think even `_po` has issue with linter

Comment (nguyentvan7) [reply libs/silly-wisher-names/src/loadTrans.ts:125]: Yea I think it is a typescript setting to allow the underscore throwaway variables. Similar thing was happening in datamine somewhere iirc. But the underscored variables are allowed in our frontend code

**Review by eeeqeee — APPROVED** [review]

**Pull request merged**